_eslintrc.txt

```diff
@@ -0,0 +1,25 @@
+{
+   "settings": {
+    "ecmascript": 5
+  },
+  "ecmaFeatures": {
+  },
+  "env": {
+    "browser": true,
+    "jquery": true
+  },
+  "rules": {
+    "semi": 1,
+    "strict": 0,
+    "indent": [2, 2],
+    "quotes": [1, "single"],
+    "no-multi-spaces": [1, {
+      "exceptions": {
+        "VariableDeclarator": true,
+        "FunctionExpression": true
+      }
+    }],
+    "key-spacing": [0, {"align": "value"}],
+    "no-underscore-dangle": 0
+  }
+}
```

Comment:
Does naming this file `_eslintrc.txt` work? Is the linter working? I usually only see this named `.eslintrc`. That's the standard, so I would recommend doing it that way.